### ISSUE TITLE
CLI Directory Lift

### ISSUE THREAD

**Issue by dOrgJelli** [Niraj-Kamdar/monowrap#7]
"Lift" the current root directory, and have a `./cli` folder which contains the `monowrap` CLI source code. This allows us to create new folders like:
- `./graph` - graph generation & solving crate (as a wrapper)
- `./run` - logic for running commands (CLI, etc)
- `./wrappers` - helpful wrappers (ex: cli-command, yarn, docker, etc)

**Comment by Niraj-Kamdar** [comment]
I agree we need refactor but honestly I am not sure how I will structure it because I want to generalize the graph solving part into its own wrapper and I know it can be useful not only here but also in other polywrap manifests like deploy, test, etc. I need to spec out more before I feel comfortable in refactoring this.
